client/my-sites/media-library/index.jsx

```diff
@@ -214,6 +222,9 @@ export default connect(
 		mediaValidationErrors: getMediaErrors( state, site?.ID ),
 		needsKeyring: needsKeyring( state, source ),
 		selectedItems: getMediaLibrarySelectedItems( state, site?.ID ),
+		isJetpack: isJetpackSite( state, site?.ID ),
+		isAtomic: isAtomicSite( state, site?.ID ),
+		hasVideoUploadFeature: hasFeature( state, site?.ID, FEATURE_VIDEO_UPLOADS ),
```

Comment:
I think `hasSiteFeature` might work better here because it checks the plan rather than the site:
```suggestion
		hasVideoUploadFeature: hasSiteFeature( site, FEATURE_VIDEO_UPLOADS ),
```

I saw it in use here: https://github.com/Automattic/wp-calypso/blob/b3095dc2cadf2aa1bb015454edeaa542b5563135/client/my-sites/site-settings/seo-settings/utils.ts#L18

Reply:
hasFeature expects a site ID so I think they do almost the same?
However I 100% agree with you. Let me commit our suggestion. Thanks 👍